examples/https.rs

```diff
@@ -1,41 +1,63 @@
 // This requires running with:
 //
 // ```bash
-// cargo run --example https --features ssl
+// cargo run --example https --features openssl-example
 // ```
 //
 // Generate a key and certificate like so:
 //
 // ```bash
-// openssl genrsa -out localhost.key 4096
-// openssl req -key localhost.key -x509 -new -days 3650 -out localhost.crt
+// openssl req -x509 -newkey rsa:4096 -nodes -keyout localhost.key -out localhost.crt -days 3650
 // ```
 
 extern crate iron;
+#[cfg(feature = "openssl-example")]
+extern crate hyper_openssl;
+#[cfg(feature = "openssl-example")]
+extern crate openssl;
 
-#[cfg(feature = "ssl")]
+#[cfg(feature = "openssl-example")]
 fn main() {
-    // Avoid unused errors due to conditional compilation ('ssl' feature is not default)
-    use iron::status;
+    // Avoid unused errors due to conditional compilation ('openssl-example' feature is not default)
+    use hyper_openssl::OpensslServer;
     use iron::{Iron, Request, Response};
-    use std::path::{Path};
-    use std::result::{Result};
+    use iron::status;
```

Comment:
Updated! I haven't published the openssl and hyper-openssl releases yet, but assuming this looks good I will.